tests/lib/cbprintf_package/testcase.yaml

```diff
@@ -1,6 +1,8 @@
 common:
   filter: CONFIG_QEMU_TARGET or CONFIG_BOARD_NATIVE_POSIX
   tags: cbprintf
+  # qemu_arc_hs6x excluded, see #38041
```

Comment:
It would be good to prefix these with `FIXME: ...` so that we can easily grep in the future.